src/tpm2_wrap.c

```diff
@@ -1726,6 +1843,30 @@ int wolfTPM2_RsaKey_WolfToTpm(WOLFTPM2_DEV* dev, RsaKey* wolfKey,
 {
     return wolfTPM2_RsaKey_WolfToTpm_ex(dev, NULL, wolfKey, tpmKey);
 }
+
+int wolfTPM2_RsaKey_PemToTpm(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* tpmKey,
+                             byte *tempBuf, int tempSz, const char *filename)
+{
+    int rc = TPM_RC_FAILURE;
+#ifndef WOLFTPM2_NO_WOLFCRYPT
```

Comment:
This also needs to check `defined(WOLFSSL_CERT_EXT) || defined(WOLFSSL_PUB_PEM_TO_DER) || defined(WOLFSSL_PEM_TO_DER)`